@here/harp-mapview/lib/MapView.ts

```diff
@@ -3078,7 +3092,27 @@ export class MapView extends THREE.EventDispatcher {
                 this.m_options.maxDistanceRatioForPoiLabels
             );
         }
-        this.m_textElementsRenderer.placeAllTileLabels();
+    }
+
+    /**
+     * Delay creation of `TextElementsRenderer` to out-of-frame to avoid fps lag.
+     */
+    private enqueueTextRendererCreation(): void {
+        if (
+            this.m_textElementsRenderer === undefined &&
+            this.m_textElementsRendererTimer === undefined
+        ) {
+            this.m_textElementsRendererTimer = setTimeout(() => {
+                delete this.m_textElementsRendererTimer;
```

Comment:
Uch, och, ech ...

I'm not sure if this statement:
> It is generally recommended to avoid using delete whenever possible
(https://stackoverflow.com/questions/43594092/slow-delete-of-object-properties-in-js-in-v8/44008788)

still stands, (intuition and common knowledge how optimizers work, say yes), but it's probably better (and current practice) to set unused properties to `undefined`, so we're not asking V8 to possibly re-evaluate shape of our object

Another guy saying the same: https://ljn.io/posts/node-performance-tips/

Minor, but i thought it's good to share.

Reply:
Sounds reasonable.